aws/rust-runtime/aws-sig-auth/src/middleware.rs

```diff
@@ -270,6 +270,7 @@ mod test {
                 .apply(req.try_clone().expect("can clone"))
                 .expect_err("no cred provider"),
         );
+        // use `new` instead of `for_tests` because the presence of a session_token will render `auth_header` below `Sensitive`.
```

Comment:
you can use `unwrap().to_string()` to debug IIRC

Reply:
Great, that worked. Thanks for the suggestion. Addressed in 851abbb.